docs/src/man/interval_hulls.md

```diff
@@ -0,0 +1,104 @@
+# Interval Hulls
+
+In this section we illustrate some interval hull operators
+as well as several plotting functionalities.
+
+```@meta
+DocTestSetup = quote
+    using LazySets, Plots, LazySets.Approximations
+end
+```
+
+## Balls and singleton plots
+
+Consider a ball in the 2-norm:
+
+```@example
```

Comment:
I need an additional `using LazySets` here, otherwise I get an error. Seems that the `DocTestSetup` above is not executed for `@example` blocks. Does it have to be an `@example` block here?

Reply:
correct; either that or i can add a label to this block, i'll use the latter.